docker/Dockerfile

```diff
@@ -3,21 +3,20 @@ FROM debian:buster-slim
 RUN set -ex; \
     apt-get update -y; \
     apt-get install -y --no-install-recommends \
-        python3 \
-        python3-ldap \
-        python3-pip \
-        python3-psycopg2 \
-        python3-setuptools \
-        python3-yaml \
+        python3=3.7.2-1 \
+        python3-ldap=3.1.0-2 \
+        python3-pip=18.1-5 \
+        python3-psycopg2=2.7.7-1 \
+        python3-setuptools=40.8.0-1 \
+        python3-yaml=3.13-2 \
     ; \
-    apt-get clean; \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; \
     :
 
 RUN set -ex; \
-    pip3 --no-cache-dir install --no-deps ldap2pg; \
+    pip3 --no-cache-dir install --no-deps ldap2pg==4.18; \
     ldap2pg --version; \
     :
 
 ENTRYPOINT ["ldap2pg"]
-WORKDIR /workspace
+WORKDIR /workspace
```

Comment:
Strange that the commit drops final newline. Is it a bug in .editorconfig ?

Reply:
It might be my Dockerfile plugin in VS-Code that removed it... or just my fondness of backspace :)

Reply:
I did not encounter any problems, but I am looking into using ldap2pg in some of our systems, and we follow best-practice in regards to version pinning.